src/examples/src/config.tsx

```diff
@@ -1659,6 +1660,11 @@ export const config = {
 					module: DirectionSpeedDial,
 					title: 'With a direction'
 				},
+				{
+					filename: 'Position',
```

Comment:
can you make this example use the iframe and sizing so it correctly displays in the example area?

```ts
  sandbox: true,
  size: 'large'
````

Reply:
same catch as me - but more helpful 👍